src/workspaces/features/WorkspaceMemberPicker/WorkspaceMemberPicker.tsx

```diff
@@ -28,8 +30,10 @@ const WorkspaceMemberPicker = (props: WorkspaceMemberPickerProps) => {
     disabled = false,
     required = false,
     withPortal = false,
+    multiple = false,
     onChange,
-    placeholder = t("Select recipients"),
+    exclude = [],
+    placeholder = t("Select member"),
   } = props;
 
   const { data, loading } = useQuery<WorkspaceMemberPickerQuery>(
```

Comment:
This component triggers two queries on mount. It's better to use a LazyQuery that would fetch the list only when the user open the select.